lib/cib/cib_ops.c

```diff
@@ -569,8 +569,8 @@ update_results(xmlNode *failed, xmlNode *target, const char *operation,
         pcmk__xe_set(xml_node, PCMK_XA_OPERATION, operation);
         pcmk__xe_set(xml_node, PCMK_XA_REASON, error_msg);
 
-        crm_warn("Action %s failed: %s (cde=%d)",
-                 operation, error_msg, return_code);
+        pcmk__warn("Action %s failed: %s (cde=%d)", operation, error_msg,
```

Comment:
While you're here, `cde` is probably supposed to be `code`.

Reply:
Adding a commit on top to address this and unindent.